src/EntityGraphQL/Schema/BaseSchemaTypeWithFields.cs

```diff
@@ -40,6 +45,35 @@ protected BaseSchemaTypeWithFields(ISchemaProvider schema, string name, string?
             RequiredAuthorization = requiredAuthorization;
         }
 
+        public void ApplyAttributes(IEnumerable<Attribute> attributes)
```

Comment:
This looks the same as the one above. Can we refactor out to a single method in a helper/extension or similar?

Reply:
couple of minor differences

* field applies validators
* One runs applyextension using IField are the other ISchemaType arg

could probably be merged, wanted to figure out a plan for the assembly scanning first.

Reply:
Ahh yes, missed those details. Worth looking at merging in some generic or other way, lots of shared code.

What is the assembly scanning happening here?

Reply:
OK I understand the scanning happening here now.

I know is is not as automatic (can have some defaults) but I feel it might be more easier to follow if we just register these? Like field extensions. 

I.e. 

```cs
schema.AddAttributeHandler<AuthorizeAttribute>();
```

That is done before any `SchemaBuilder` reflection, then it might be easier to follow what is happening.

Can register a few by default - Obsolete, etc.

Thoughts?

Reply:
yeah actually i quite like that idea.